scripts/buildx

```diff
@@ -23,11 +23,11 @@ if [ "$TARGETOS" != "linux" ]; then
     OPT_OS="-${TARGETOS}"
 fi
 
-if [ "$TARGETOS" = "Linux" ]; then
-    OTHER_LINKFLAGS="-extldflags -static -s"
+if [ "$TARGETOS" = "linux" ]; then
```

Comment:
this needs to be fixed in `scripts/build` as well

Reply:
Fixed!